src/ol/collection.js

```diff
@@ -82,7 +82,7 @@ ol.CollectionProperty = {
  */
 ol.Collection = function(opt_array) {
 
-  goog.base(this);
+  ol.Object.call(this);
```

Comment:
Wouldn't `goog.base(this)` (single argument) equivalent to `ol.Object.apply(this, arguments)` rather than `ol.Object.call(this)`?